docs/_Docs_GetStarted.md

```diff
@@ -68,78 +73,79 @@ The final step for installation is copying over some of the files from our submo
 
-## Reputation Miner
+For simple interactions, [Remix](http://remix-alpha.ethereum.org/) is a good lightweight way to call specific functions and get network information from the contracts.
 
-The reputation mining client is usable for testing but it has limited functionality. It currently has no support for the challenge-response process to accommodate multiple submitted reputation root hashes. Still, it is possible to run a single miner instance for usable reputation scores on a local test network.
-
-### Start Mining Client
-
-You can start the reputation mining client using the following command.
+Rather than import the entire set of contracts into remix, use the included `solidity-steamroller` to flatten the needed interface contracts to the `build/flattened/` directory:
 
 ```
-node packages/reputation-miner/bin/index.js --colonyNetworkAddress { COLONYNETWORK_ADDRESS } --minerAddress { MINER_ADDRESS }
+$ yarn flatten:contracts
 ```
 
-The `minerAddress` in the execution above is the sixth account in `ganache-accounts.json` if running locally.
-
-The `colonyNetwork` address in the execution above is not the address outputted at contract deployment, but is the address of the Colony Network `EtherRouter`. See [Upgrades to the Colony Network](/colonynetwork/docs-the-delegate-proxy-pattern/) for more information about the EtherRouter design pattern.
-
-### Force Reputation Updates
+Alternatively, you can put them into the directory of your choice, e.g. `~/Downloads/`:
+```
+$ yarn steamroller /contracts/IColonyNetwork.sol > ~/Downloads/flatIColonyNetwork.sol
+```
 
-The client is set to provide a reputation update once per hour. For testing, you'll likely want to "fast-forward" your network through a few submissions to see usable reputation.
+Import the flattened contract into the remix IDE, and then call `createColony()`
 
-You can move the network forward by an hour with the following command.
+### Access with the Truffle console
 
+First, add a private key of your choice to the `truffle.js` configuration file:
 ```
-curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[86400],"id": 1}' localhost:8545
+goerli: {
+      provider: () => {
+        return new HDWalletProvider("replace-with-private-key-when-using", "https://goerli.infura.io/v3/e21146aa267845a2b7b4da025178196d");
+      },
+      network_id: "5"
+    }
+  },
 ```
 
-Once you have moved the network forward by an hour, you can then mine a new block with the following command.
-
+Then, start up the truffle console and connect to testnet:
 ```
-curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_mine","params":[]}' localhost:8545
+$ yarn truffle console --network goerli
 ```
+In the truffle console, instantiate the IColonyNetwork interface for `glider-rc.1`:
+```
+truffle(goerli)> let IColonyNetwork = await IColonyNetwork.at("0x79073fc2117dD054FCEdaCad1E7018C9CbE3ec0B")
```

Comment:
We should avoid naming local variables as truffle artefacts. `let IColonyNetwork` -> `let colonyNetwork`